### DISCUSSION TITLE
Build SFCs + defineAsyncComponent

### DISCUSSION THREAD

**Discussion by SebastiaoAlterdata** [vuejs/rfcs#683] [General]
**What problem does this feature solve?**

I have a SFC and when I'm running the project using **defineAsyncComponent** variables it works fine, but when I'm try to build the C.E, it was bad and get a build error.

```
"vue": "^3.4.33"
"vite": "^5.3.4"
"@vitejs/plugin-vue": "^5.0.5"
```
```
const myComponent = defineAsyncComponent(
    () => import('@components/MyComponent.vue')
  );
```

![image](https://github.com/user-attachments/assets/5dc3d790-c42c-45a5-91e0-2df6c9a83135)

**Comment by posva** [comment]
Remember to use the [Discord chat](https://chat.vuejs.org/) to ask questions!
